temporal/api/cloud/connectivityrule/v1/message.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+
+package temporal.api.cloud.connectivityrule.v1;
+
+option go_package = "go.temporal.io/api/cloud/connectivityrule/v1;connectivityrule";
+option java_package = "io.temporal.api.cloud.connectivityrule.v1";
+option java_multiple_files = true;
+option java_outer_classname = "MessageProto";
+option ruby_package = "Temporalio::Api::Cloud::ConnectivityRule::V1";
+option csharp_namespace = "Temporalio.Api.Cloud.ConnectivityRule.V1";
+
+import "temporal/api/cloud/resource/v1/message.proto";
+import "google/protobuf/timestamp.proto";
+
+message ConnectivityRule {
+  // The id of the private connectivity rule.
+  string id = 1;
+
+  // The connectivity rule specification.
+  ConnectivityRuleSpec spec = 2;
+
+  // The current version of the connectivity rule specification.
+  // The next update operation will have to include this version.
+  string resource_version = 3;
+  temporal.api.cloud.resource.v1.ResourceState state = 4;
```

Comment:
New lines are a bit off. 
There is a extra line between id and spec but not between resource_version and state.